modules/internal/DefaultSparse.chpl

```diff
@@ -423,7 +423,7 @@ module DefaultSparse {
     }
 
     proc dsiTargetLocales() {
-      compilerError("targetLocales is unsuppported by sparse domains");
+      compilerError("targetLocales is unsupported by sparse domains");
```

Comment:
Do you know if there are tests that intentionally trigger this error that would need to be updated?